src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -261,10 +261,10 @@ def user_and_role(user, role = nil, options = {})
       icon = user_icon(user)
     end
 
-    if !(real_name.empty? || opt[:short])
-      printed_name = "#{real_name} (#{user})"
-    else
+    if real_name.empty? || opt[:short]
```

Comment:
I was discussing something related to that with @mschnitzer and he changed that to check something. So I would say that he commited it by accident.